packages/eslint-plugin/src/configs/react.js

```diff
@@ -273,15 +313,24 @@ const config = {
      * @see https://github.com/import-js/eslint-plugin-import
      */
     'import/no-extraneous-dependencies': ['error', { devDependencies: false }],
+    'import/no-duplicates': 'warn',
```

Comment:
Aren't the new rules going to cause a bunch of spam in build logs due to existing violations? And I still don't really see the point of even having rules if you can just ignore the warnings (especially since it creates a nuisance for others running lint because they see warnings from code they didn't even touch).